nl_NL/LC_MESSAGES/youtube.po

```diff
@@ -4,191 +4,191 @@ msgstr ""
+msgstr "Deze tekst wordt weergegeven wanneer kijkers in de beschrijving van je video op 'Meer weergeven' klikken."
 
 msgid "TAGS"
 msgstr "Tags"
 
 msgid "NONE"
-msgstr "None"
+msgstr "Geen"
 
 msgid "YOUTUBE_LOOP_COUNT"
-msgstr "Number of plays"
+msgstr "Hoeveelheid loops"
 
 msgid "YOUTUBE_LOOP_COUNT_EXPLANATION"
-msgstr "Number of times a looped Flipnote will repeat. If 1, the Flipnote will only play once."
+msgstr "Aantal keren dat een Flipnote in een lus wordt herhaald. Indien 1 zal de Flipnote maar één keer worden afgespeeld."
 
 msgid "YOUTUBE_VISIBILITY"
-msgstr "Visibility"
+msgstr "Zichtbaarheid"
 
 msgid "YOUTUBE_VISIBILITY_PUBLIC"
-msgstr "Public"
+msgstr "openbaar"
 
 msgid "YOUTUBE_VISIBILITY_UNLISTED"
-msgstr "Unlisted"
+msgstr "Verborgen"
 
 msgid "YOUTUBE_VISIBILITY_PRIVATE"
-msgstr "Private"
+msgstr "privé"
 
 msgid "SUBMIT"
-msgstr "Submit"
+msgstr "Verstuur"
 
 # Upload status
 msgid "UPLOAD_STATUS"
 msgstr "Upload Status"
 
 msgid "YOUTUBE_PENDING_UPLOAD"
-msgstr "Your Flipnote is currently waiting to be uploaded. Check back in a few minutes."
+msgstr "Je flipnote is op dit moment aan het wachten om geupload te worden. kijk over een paar minuutjes opnieuw"
 
 # Position in queue: 4
 msgid "POSITION_IN_QUEUE"
-msgstr "Position in queue: %d"
+msgstr "Positie in wachtrij: %d"
 
 msgid "THEATRE_STATUS_TRACKER"
-msgstr "Watch your live Upload Tracker here:"
+msgstr "Bekijk je live upload tracker hier"
 
 msgid "REVIEW_DETAILS_BEFORE_UPLOAD"
-msgstr "Please review these options before starting your upload."
+msgstr "Bekijk deze opties voordat je je flipnote gaat uploaden"
 
 # ...do so, visit https://flipnot.es/ABC123/yt
 msgid "CHANGE_OPTIONS_ON_THEATRE"
-msgstr "You can change these options before upload. To do so, visit:"
+msgstr "Je kunt deze opties wijzigen voordat je je flipnote upload om dat te doen ga naar:"
 
 # ...please visit: https://flipnot.es/ytsetup
 msgid "SET_DEFAULTS_ON_THEATRE"
-msgstr "To change your default upload options, please visit:"
+msgstr "Om je standaart opties te wijzigen, ga naar:"
 
 msgid "STATUS"
 msgstr "Status"
 
 msgid "YOUTUBE_STATUS_NEW"
-msgstr "New"
+msgstr "Nieuw!"
 
 msgid "YOUTUBE_STATUS_PROCESSING"
-msgstr "Processing"
+msgstr "Bezig"
 
 msgid "YOUTUBE_STATUS_SUCCESS"
-msgstr "Done!"
+msgstr "Klaar!"
 
 msgid "YOUTUBE_STATUS_PAUSED"
-msgstr "Paused"
+msgstr "gepauzeerd"
 
 msgid "YOUTUBE_STATUS_ERROR"
 msgstr "Error"
 
 msgid "YOUTUBE_STATUS_UNKNOWN"
-msgstr "Unknown"
+msgstr "Onbekend"
 
 msgid "YOUTUBE_STATUS_WAITING_TO_BE_PROCESSED"
-msgstr "Your upload is currently waiting to be processed."
+msgstr "Je upload wacht momenteel om te worden verwerkt."
 
 msgid "YOUTUBE_STATUS_BEING_PROCESSED"
-msgstr "Your upload is currently being processed."
+msgstr "Je upload word op dit moment verwerkt!"
 
 msgid "YOUTUBE_STATUS_COMPLETE"
-msgstr "Your upload is complete!"
+msgstr "Your upload is Klaar!"
```

Comment:
This doesn't seem entirely translated based on the translated strings above, and "Klaar" doesn't seem like it should have the first letter capitalized.